galley/docker/Dockerfile.galley

```diff
@@ -10,4 +10,15 @@ FROM gcr.io/distroless/static as distroless
 # This will build the final image based on either default or distroless from above
 FROM ${BASE_DISTRIBUTION}
 ADD galley /usr/local/bin/
+
+# 1001 is chosen as the UID to be added to the istio group as this is the UID that galley in the docker container runs as.
```

Comment:
why special uid? do we need to specify it explicitly?

Reply:
I have googled and found that 
> With the update to 2.0, the Docker container was modified so that Prometheus runs as the user "nobody" (UID/GID 99/99) (#2859). While it's good that Prometheus is no longer running as root, it's not the best idea either to run as "nobody". There are many examples of Docker containers out there that allow changing the UID/GID at runtime that could be borrowed from. At a minimum a different static UID/GID should be used.

append the original issue - https://github.com/prometheus/prometheus/issues/3441
also we can see the discussion in grafana - https://github.com/grafana/grafana-docker/pull/146/commits/e2872ea0deac347fd0a9d25baf7c70683aa94a5a#r177055710

Reply:
>  There are many examples of Docker containers out there that allow changing the UID/GID at runtime that could be borrowed from.

It seems we could do that? I am not an expert in this area though